### PR TITLE
Add chargebee subscription cancellation handler

### PR DESCRIPTION
**Pull request by AvoidMe**

### CODE REVIEW

File: modules/chargebee-integration/app/kuberlogic.go [hunk 1]
```diff
@@ -41,7 +44,124 @@ func createService(logger *zap.SugaredLogger, svc *models.Service) error {
...
+	}
+
+	response, err := apiClient.Service.ServiceList(
+		params,
+		httptransport.APIKeyAuth("X-Token", "header", viper.GetString("KUBERLOGIC_APISERVER_TOKEN")),
+	)
+	if err != nil {
+		return nil, err
+	}
+	if len(response.Payload) > 0 {
```
Comment (ynnt) [inline modules/chargebee-integration/app/kuberlogic.go:78]: It's worth to fail when we have more than one service

File: modules/chargebee-integration/app/webhook.go [hunk 2]
```diff
@@ -32,46 +33,90 @@ func WebhookHandler(baseLogger *zap.SugaredLogger, mapping []map[string]string)
...
+
+func handleSubscriptionCancelled(logger *zap.SugaredLogger, event map[string]interface{}, mapping []map[string]string) {
+	subscriptionId, err := GetSubscriptionId(event)
+	if err != nil {
+		logger.Error("Error extracting subscription from payload", err)
+		return
+	}
+	logger = logger.With("subscription id", subscriptionId)
+	// Check if service exists
+	service, err := getServiceBySubscriptionId(logger, subscriptionId)
```
Comment (ynnt) [inline modules/chargebee-integration/app/webhook.go:89]: We need to retry when something wrong happens during this step: there might be an issue getting service even when it exists.

**Review by ynnt — CHANGES_REQUESTED** [review]

File: modules/chargebee-integration/app/kuberlogic.go (hunk 1 above)
Comment (AvoidMe) [reply modules/chargebee-integration/app/kuberlogic.go:78]: We can have multiple services with single subscription id?

File: modules/chargebee-integration/app/webhook.go (hunk 2 above)
Comment (AvoidMe) [reply modules/chargebee-integration/app/webhook.go]: Fixed

File: modules/chargebee-integration/app/kuberlogic.go [hunk 3]
```diff
@@ -41,7 +44,124 @@ func createService(logger *zap.SugaredLogger, svc *models.Service) error {
...
+	)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func waitForBackup(logger *zap.SugaredLogger, serviceID string, backupID string) error {
+	timeout := time.Second
+	for i := 13; i > 0; i-- {
```
Comment (telepenin) [inline modules/chargebee-integration/app/kuberlogic.go:141]: 13 - magic numbers. Is it retries? Could you add comments or rename to constant?

**Review by telepenin — APPROVED** [review]

**Pull request merged**